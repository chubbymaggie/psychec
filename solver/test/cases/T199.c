void f() {
    if (a == 10) {}

    {
        switch (b) {
        case a: break;
        case b: break;
        default: break;
        }
    }
}
