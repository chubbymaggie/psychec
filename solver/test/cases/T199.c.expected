/* Forward declarations */

/* Type definitions */

/* Variables and functions */
#define  a 129 
#define  b 128 

